engine/schema/src/com/cloud/network/dao/NetworkVO.java

```diff
@@ -616,7 +616,8 @@ public void setStrechedL2Network(boolean strechedL2Network) {
         this.strechedL2Network = strechedL2Network;
     }
 
-    public void setVpcId(Long vpcId) {
-        this.vpcId = vpcId;
-    }
+    public void setVpcId(Long vpcId) { this.vpcId = vpcId;}
+
+    public void setIsReduntant(boolean reduntant){this.isRedundant = reduntant;}
```

Comment:
I think this fragment should look like
    public void setVpcId(Long vpcId) {
        this.vpcId = vpcId;
    }

    public void setIsReduntant(boolean reduntant) {
        this.isRedundant = reduntant;
    }
Why did you change the format of the existing function this way. It does not adhere to best practice standard code formatting rules.

please change this. we need a force push and rebuild anyway to see a jenkins output for this old PR